packages/fiori/src/Wizard.ts

```diff
@@ -613,20 +622,23 @@ class Wizard extends UI5Element {
 		responsivePopover.showAt(oDomTarget);
 	}
 
-	async _onGroupedTabClick(event) {
-		if (this._isGroupAtStart(event.target)) {
-			return this._showPopover(event.target, true);
+	async _onGroupedTabClick(e: MouseEvent) {
+		const eTarget: WizardTab = e.target as WizardTab;
+
+		if (this._isGroupAtStart(eTarget)) {
+			return this._showPopover(eTarget, true);
 		}
 
-		if (this._isGroupAtEnd(event.target)) {
-			return this._showPopover(event.target, false);
+		if (this._isGroupAtEnd(eTarget)) {
+			return this._showPopover(eTarget, false);
 		}
 	}
 
-	_onOverflowStepButtonClick(event) {
-		const tabs = Array.from(this.shadowRoot.querySelectorAll("[ui5-wizard-tab]"));
-		const stepRefId = event.target.getAttribute("data-ui5-header-tab-ref-id");
-		const stepToSelect = this.slottedSteps[stepRefId - 1];
+	_onOverflowStepButtonClick(e: MouseEvent) {
+		const tabs: Array<WizardTab> = Array.from(this.shadowRoot!.querySelectorAll("[ui5-wizard-tab]"));
```

Comment:
Array.from(this.shadowRoot!.querySelectorAll<WizardStep>("[ui5-wizard-tab]"));

Reply:
nor this